### ISSUE TITLE
Enable download by chapter

### ISSUE THREAD

**Issue by Waffle555** [houtianze/libby-audiobook-exporter-browser-extension#4]
Hello, thanks for developing this extension! Do you think it's possible to have the files be split into chapters? What would it take from your perspective? Thank you!

**Comment by DIntriglia** [comment]
Based on the openbook.json file, it looks like this should be possible as that has the breakdown of each chapter/context of each file. It would probably need to be done after the download, but a python script should be able to concatenate the files, and then split them into the audiobook. Thoughts?

**Comment by houtianze** [comment]
Yes, @DIntriglia I agree that I think it's doable using the metadata inside `openbook.json` but for now I'm not very inclined to do it:
1. Tim/effort
2. Most music/audio book player can just continue playing the mp3 files once you throw them in, so though this chapter split is quite nice to have, but I don't feel it's something very critical.
